pagerduty/resource_pagerduty_event_orchestration_path_global.go

```diff
@@ -0,0 +1,423 @@
+
+func buildEventOrchestrationPathGlobalRuleActionsSchema() map[string]*schema.Schema {
+	a := eventOrchestrationPathGlobalCatchAllActionsSchema
+	a["route_to"] = &schema.Schema{
+		Type:     schema.TypeString,
+		Optional: true,
+	}
+
+	return a
+}
+
+func resourcePagerDutyEventOrchestrationPathGlobal() *schema.Resource {
+	return &schema.Resource{
+		ReadContext:   resourcePagerDutyEventOrchestrationPathGlobalRead,
+		CreateContext: resourcePagerDutyEventOrchestrationPathGlobalCreate,
+		UpdateContext: resourcePagerDutyEventOrchestrationPathGlobalUpdate,
+		DeleteContext: resourcePagerDutyEventOrchestrationPathGlobalDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourcePagerDutyEventOrchestrationPathGlobalImport,
+		},
+		CustomizeDiff: checkExtractions,
+		Schema: map[string]*schema.Schema{
+			"event_orchestration": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"set": {
+				Type:     schema.TypeList,
+				Required: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"rule": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"id": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"label": {
+										Type:     schema.TypeString,
+										Optional: true,
+									},
+									"condition": {
+										Type:     schema.TypeList,
+										Optional: true,
+										Elem: &schema.Resource{
+											Schema: eventOrchestrationPathConditionsSchema,
+										},
+									},
+									"actions": {
+										Type:     schema.TypeList,
+										Required: true,
+										MaxItems: 1,
+										Elem: &schema.Resource{
+											Schema: eventOrchestrationPathGlobalRuleActionsSchema,
+										},
+									},
+									"disabled": {
+										Type:     schema.TypeBool,
+										Optional: true,
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			"catch_all": {
+				Type:     schema.TypeList,
+				Required: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"actions": {
+							Type:     schema.TypeList,
+							Required: true,
+							MaxItems: 1,
+							Elem: &schema.Resource{
+								Schema: eventOrchestrationPathGlobalCatchAllActionsSchema,
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func resourcePagerDutyEventOrchestrationPathGlobalRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	var diags diag.Diagnostics
+
+	client, err := meta.(*Config).Client()
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	retryErr := resource.Retry(2*time.Minute, func() *resource.RetryError {
+		id := d.Id()
+		t := "global"
+		log.Printf("[INFO] Reading PagerDuty Event Orchestration Path of type %s for orchestration: %s", t, id)
```

Comment:
**Not a blocker:** Comparing this file with the `*_path_service.go` equivalent made me realize we've got a little typo in the "Reading PagerDuty Event Orchestration Path" log line for a Service Orchestration! Currently the log would read "of type service for orchestration: POOPBUG" ([here](https://github.com/PagerDuty/terraform-provider-pagerduty/blob/a6e7827bd4c268dd76c687a8e2e1918fd7c52c3b/pagerduty/resource_pagerduty_event_orchestration_path_service.go#L184)) but it should *probably* read "of type service for service: POOPBUG" since the ID is the the Service that the Orchestration Path belongs to.

Reply:
Good catch, updated the message in [4f12ba4](https://github.com/PagerDuty/terraform-provider-pagerduty/pull/622/commits/4f12ba4ef2e054b95c5d738ee2506423c1b54d6a)